be/src/exec/base_scanner.cpp

```diff
@@ -179,4 +179,21 @@ bool BaseScanner::fill_dest_tuple(const Slice& line, Tuple* dest_tuple, MemPool*
     }
     return true;
 }
+
+void BaseScanner::fill_slots_of_columns_from_path(int start, const std::vector<std::string>& columns_from_path) {
+    if (start <= 0) {
```

Comment:
I think this check is useless

Reply:
> I think this check is useless

But we should skip the case of StreamLoadTask